test/helpers/helpers.js

```diff
@@ -50,20 +74,31 @@ function closeAll(server, statsd, allowErrors, done) {
  * Returns all permutations of test types to run through
  */
 function testTypes() {
-  return [[`${UDP} ${CLIENT}`, UDP, CLIENT, UDP_METRIC_END],
+  const testTypesArr = [[`${UDP} ${CLIENT}`, UDP, CLIENT, UDP_METRIC_END],
     [`${UDP} ${CHILD_CLIENT}`, UDP, CHILD_CLIENT, UDP_METRIC_END],
     [`${UDP} ${CHILD_CHILD_CLIENT}`, UDP, CHILD_CHILD_CLIENT, UDP_METRIC_END],
     [`${TCP} ${CLIENT}`, TCP, CLIENT, TCP_METRIC_END],
-    [`${TCP} ${CHILD_CLIENT}`, TCP, CHILD_CLIENT, TCP_METRIC_END],
-    [`${TCP} ${CHILD_CHILD_CLIENT}`, TCP, CHILD_CHILD_CLIENT, TCP_METRIC_END]];
+    [`${TCP} ${CHILD_CLIENT}`, TCP, CHILD_CLIENT, TCP_METRIC_END]];
```

Comment:
Only keep one child_child_client, as that never has tested much.  And then add UDS tests when applicable to all below, which makes sure we test this new functionality through the full test suite